eclipse/oxygen/gcp-eclipse-oxygen.target

```diff
@@ -24,7 +24,6 @@
       <unit id="org.eclipse.jetty.servlet" version="9.4.0.v20161208"/>
       <unit id="org.eclipse.jetty.server" version="9.4.0.v20161208"/>
       <unit id="org.eclipse.jetty.util" version="9.4.0.v20161208"/>
-      <unit id="ch.qos.logback.slf4j" version="1.0.7.v201505121915"/>
```

Comment:
No, but this dependency shouldn't have been here as I previously moved Logback to be resolved from the Orbit repository (which you can see further below).